barretenberg/cpp/src/barretenberg/stdlib/primitives/field/field_conversion.hpp

```diff
@@ -78,7 +78,7 @@ template <typename Builder, typename T> T convert_from_bn254_frs(Builder& builde
         return fr_vec[0];
     } else if constexpr (IsAnyOf<T, fq<Builder>>) {
         ASSERT(fr_vec.size() == 2);
-        fq<Builder> result(fr_vec[0], fr_vec[1], 0, 0);
+        fq<Builder> result(fr_vec[0], fr_vec[1]);
```

Comment:
I had a hunch that this should be better cus otherwise we might have to do some constraining magic of the 0s, let me know if I am wrong

Reply:
It's the same. The default values for the constructor are false and zero

Reply:
Just a bit cleaner